platforms/windows/hax_wrapper.c

```diff
@@ -237,47 +226,57 @@ void hax_disable_irq(void)
     asm_disable_irq();
 }
 
-void hax_error(char *fmt, ...)
+int hax_log_level(int level, const char *fmt,  ...)
 {
     va_list arglist;
+    va_start(arglist, fmt);
 
+    if (level >= HAX_LOG_DEFAULT)
+        vDbgPrintExWithPrefix("haxm: ", DPFLTR_IHVDRIVER_ID, DPFLTR_ERROR_LEVEL, fmt, arglist);
```

Comment:
`va_end()` missed. Even if it is dummy on Windows.

Reply:
That's right, it's better pair va_start() with va_end() even it's dummy. Thanks for point it out.